release/gcb/local.sh

```diff
@@ -37,8 +36,20 @@ popd || return
 
 pushd "${ROOT}" || exit
   create_manifest_check_consistency "${artifacts}/manifest.txt"
-  make_istio "${artifacts}" "${DOCKER_HUB}" "${DOCKER_HUB}" "${NEW_VERSION}" "${BRANCH}"
+  make_istio "${artifacts}" "${DOCKER_HUB}" "${DOCKER_HUB}" "${NEW_VERSION}" "${CNI_BRANCH}"
 popd || return
 
 docker_tag_images  "${DOCKER_HUB}" "${NEW_VERSION}" "${artifacts}"
 docker_push_images "${DOCKER_HUB}" "${NEW_VERSION}" "${artifacts}"
+
+pushd "${artifacts}" || exit
+  fix_values_yaml "${NEW_VERSION}" "${DOCKER_HUB}"
```

Comment:
This calls gsutil it looks like?

Reply:
No, it is function in gcb_lib.sh file line 146

Reply:
but that function calls gsutil I think